src/savegame.h

```diff
@@ -0,0 +1,229 @@
+#ifndef SAVEGAME_H_
+#define SAVEGAME_H_
+
+#include "stream.h"
+#include <vector>
+
+const int HEROES_DATA_OFFSET 		= 0x019a9b;
```

Comment:
These magic file offsets don't need to be visible to the whole game code. Putting them in the CPP file would be better.